rascaline/tests/lode-madelung.rs

```diff
@@ -0,0 +1,103 @@
+use approx::assert_relative_eq;
+use rascaline::calculators::lode::LodeRadialBasis;
+use rascaline::calculators::{LodeSphericalExpansionParameters, CalculatorBase, LodeSphericalExpansion};
+use rascaline::systems::{System, SimpleSystem, UnitCell};
+use rascaline::{Calculator, Matrix3, Vector3D, CalculationOptions};
+
```

Comment:
Might be worth adding a small explanation of what's going on here, what's a Madelung constant for people not familliar with it, and how is it related to lode